src/Components/CreateImageWizard/formComponents/FileSystemConfiguration.js

```diff
@@ -83,9 +83,12 @@ const FileSystemConfiguration = ({ ...props }) => {
     if (customizations && customizations.filesystem && isSuccess) {
       const newRows = rows;
 
-      // Only keep the required partitions that have a valid mount points
+      // Only keep the required partitions that have a mountpoint which root
+      // directory belongs to the supported ones.
```

Comment:
Had to read the comment a few times before understanding it, maybe "that have a mountpoint _of which the_ root directory belongs to the supported ones?"

Reply:
Maybe something simpler?

Filter out unsupported mountpoints, keeping the root mountpoint

Reply:
Yeah that comment is wrong :sweat_smile: Thanks for catching that. What about:

`The filesystems customizations required by the OpenSCAP profile may contain some incompatible mountpoints. They need to be filtered out to avoid any UX rendering issues.`

Reply:
Lol spending a lot of time discussing a comment 

`The filesystem customizations required by the OpenSCAP profile may contain some unsupported mountpoints. They need to be filtered out to prevent undefined behaviour in the frontend and the backend`

Since that could cause failed composes if submitted to the backend too.

Reply:
I love it! Thanks :smile:

Reply:
> Lol spending a lot of time discussing a comment

Yeah that's true. Good comments are hard to write though.

Reply:
I mean it would be equally valid to leave the comment out here since it is pretty clear what the code is doing :wink: